main.js

```diff
@@ -0,0 +1,98 @@
+
+let clear = document.querySelector('.clr')
+clear.addEventListener("click", function(){
+    display.textContent = ""
+})
+
+let number1 = document.querySelector('.num1')
+number1.addEventListener("click", function (){
+    display.textContent += "1"
+    console.log(1)
+})
+
+let number2 = document.querySelector('.num2')
+number2.addEventListener("click", function (){
+    display.textContent += "2"
+})
+
+let number3 = document.querySelector('.num3')
+number3.addEventListener("click", function (){
+    display.textContent += "3"
+})
```

Comment:
Consider how these are almost identical. What could you do to eliminate duplication next time?